pkg/apiserver/apiserver_test.go

```diff
@@ -157,6 +157,16 @@ func tlsSecretGetAction() core.Action {
 		apiCertSecretName)
 }
 
+func cdiConfigGetAction() core.Action {
```

Comment:
maybe change to `cdiConfigMapGetAction`?
I'm adding `CDIConfig` CR now so naming is confusing.

Reply:
Sounds good